composer/algorithms/cutmix/cutmix.py

```diff
@@ -14,7 +14,7 @@
 
 from composer.core import Algorithm, Event, State
 from composer.loggers import Logger
-from composer.models.loss import _check_for_index_targets
+from composer.loss.loss import _check_for_index_targets
```

Comment:
Curious if the loss functions are specific to the models, and if so, would it make sense for it to be something like `composer.models.loss` instead? Thinking it would be nice to be parallel to how schedulers are part of composer.optim. I'm hesitant to add a high level folder here, but can be convinced if this is what other frameworks do or the loss function is uncoupled from the models.

Reply:
In addition, `_check_for_index_targets` should be a public function, if it is being imported by another module. I know it's tangential to this PR, but would you be able to clean that up here?

Reply:
loss functions are used by models but they aren't specific to individual models. for example: cross_entropy can be used for almost any of our models.

will change `check_for_index_targets` back. I changed it during the doc update because i thought it was specific to soft_cross_entropy only. Though maybe it belongs in some sort of util file.